span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/util/AttributeValueCreator.java

```diff
@@ -28,4 +29,22 @@ public static AttributeValue createFromJaegerKeyValue(JaegerSpanInternalModel.Ke
 
     return valueBuilder.build();
   }
+
+  public static JaegerSpanInternalModel.KeyValue convertAttributeToKeyValue(
```

Comment:
nit : Should we move to another file? This one looks more like a converter.

Reply:
Ig this is resolved in the above comment. btw what does nit mean over here @kotharironak ?

Reply:
Quoting from so, 
```
Sometimes the reviewer will prefix his comments with "Nit:". This means that he's just "nitpicking"--you don't have to fix these points, but we'd like you to.
```
https://stackoverflow.com/questions/27810522/what-does-nit-mean-in-hacker-speak